2023-07-31 14:23:34,406 INFO    MainThread:11533 [wandb_setup.py:_flush():76] Current SDK version is 0.15.4
2023-07-31 14:23:34,406 INFO    MainThread:11533 [wandb_setup.py:_flush():76] Configure stats pid to 11533
2023-07-31 14:23:34,406 INFO    MainThread:11533 [wandb_setup.py:_flush():76] Loading settings from /home/bugsie/.config/wandb/settings
2023-07-31 14:23:34,406 INFO    MainThread:11533 [wandb_setup.py:_flush():76] Loading settings from /home/bugsie/PycharmProjects/FiberPhotometryAnalyzer/wandb/settings
2023-07-31 14:23:34,406 INFO    MainThread:11533 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-07-31 14:23:34,406 INFO    MainThread:11533 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-07-31 14:23:34,406 INFO    MainThread:11533 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'train.py', 'program': '/home/bugsie/PycharmProjects/FiberPhotometryAnalyzer/train.py'}
2023-07-31 14:23:34,406 INFO    MainThread:11533 [wandb_init.py:_log_setup():507] Logging user logs to /home/bugsie/PycharmProjects/FiberPhotometryAnalyzer/wandb/run-20230731_142334-r1np9nd2/logs/debug.log
2023-07-31 14:23:34,406 INFO    MainThread:11533 [wandb_init.py:_log_setup():508] Logging internal logs to /home/bugsie/PycharmProjects/FiberPhotometryAnalyzer/wandb/run-20230731_142334-r1np9nd2/logs/debug-internal.log
2023-07-31 14:23:34,406 INFO    MainThread:11533 [wandb_init.py:init():547] calling init triggers
2023-07-31 14:23:34,406 INFO    MainThread:11533 [wandb_init.py:init():554] wandb.init called with sweep_config: {}
config: {}
2023-07-31 14:23:34,406 INFO    MainThread:11533 [wandb_init.py:init():591] wandb.init() called when a run is still active
2023-07-31 14:23:40,833 INFO    MainThread:11533 [wandb_run.py:_config_callback():1283] config_cb None None {'input_shape': [1, 1000, 1], 'embedding_shape': [1, 16], 'learning_rate': 0.0001, 'num_layers': 3, 'num_heads': 8}
2023-07-31 14:54:16,049 WARNING MsgRouterThr:11533 [router.py:message_loop():77] message_loop has been closed
